packages/plans-grid-next/src/hooks/data-store/use-plan-billing-description.tsx

```diff
@@ -224,7 +232,7 @@ export default function usePlanBillingDescription( {
 							'then %(rawPrice)s billed annually, excl. taxes',
 						{
 							args: {
-								introOfferFormattedPrice: introOffer.formattedPrice,
+								introOfferFormattedPrice: introOfferFullTermText,
```

Comment:
`Non-blocking / Nitpick:`

I noticed that you marked the introOffer.formattedPrice field [as deprecated](https://github.com/Automattic/wp-calypso/pull/95428/files#diff-d822933ae1c209e261cde32fc0c43cf346c6271be748990c5c5edd5c289f48bcR70-R73).

I did a cursory search and didn't see other usages of `introOffer.formattedPrice` ( it was fast so not 100% certain ), so we might be able to remove the `formattedPrice` field entirely. Not important for this PR, but wondering what your thoughts are @chriskmnds